pttrack/templates/pttrack/document_detail.html

```diff
@@ -28,10 +28,15 @@
     <small class="text-muted">uploaded by {{ document.author }} on {{ document.written_datetime }}</small>
     {% if document.last_modified != document.written_datetime %}
     <br><small class="text-muted">last modified {{document.last_modified}}</small>
-    {% endif %}</p>
+    {% endif %}
+  <div class="row">
+    <h3><br>&nbsp;&nbsp;<a href="{{ document.image.url }}">Link to File</a></h3>
+    <small class="text-muted">&nbsp;&nbsp;&nbsp;&nbsp;(Preview and Comments Below)</small>
+  </div>
   <div class="row ">
-    <img class="img-rounded img-responsive center-block" src="{{ document.image.url }}" alt="alt_text">
+    <img class="img-rounded img-responsive center-block" src="{{ document.image.url }}" alt="Please Click Link Below">
```

Comment:
Oh! You may be right--I am honestly not sure what the default behavior is, and it's likely dependent. If Chrome and Firefox both do it, that's probably good enough for now.